arch/risc-v/src/common/riscv_internal.h

```diff
@@ -87,16 +87,16 @@
 /* Format output with register width and hex */
 
 #ifdef CONFIG_ARCH_RV32
```

Comment:
should we check CONFIG_ARCH_RV64ILP32 too?

Reply:
seems no need. uintreg_t of rv64ilp32 is also 64 bit, and ARCH_RV64ILP32 depends on ARCH_RV64.